databricks/provider.go

```diff
@@ -197,6 +218,41 @@ func providerConfigureAzureClient(d *schema.ResourceData, providerVersion string
 	return &dbClient, nil
 }
 
+// tryDatabricksCliConfigFile sets Host and Token from ~/.databrickscfg file if it exists
+func tryDatabricksCliConfigFile(d *schema.ResourceData, config *service.DBApiClientConfig) error {
+	configFile, err := homedir.Expand(d.Get("config_file").(string))
+	if err != nil {
+		return err
+	}
+	cfg, err := ini.Load(configFile)
+	if err != nil {
+		return fmt.Errorf("Authentication is not configured for provider. Please configure it\n"+
+			"through one of the following options:\n"+
+			"1. DATABRICKS_HOST + DATABRICKS_TOKEN environment variables.\n"+
+			"2. host + token provider argumeents.\n"+
+			"3. Run `databricks configure --token` that will create %s file.\n\n"+
+			"Please check https://bit.ly/2XCtuZU for details", configFile)
+	}
+	if profile, ok := d.GetOk("profile"); ok {
+		dbcliConfig := cfg.Section(profile.(string))
+		token := dbcliConfig.Key("token").String()
+		if "" == token {
+			return fmt.Errorf("Config file %s is corrupt: cannot find token in %s profile",
+				configFile, profile)
+		}
+		config.Token = token
+
+		host := dbcliConfig.Key("host").String()
+		if "" == host {
+			return fmt.Errorf("Config file %s is corrupt: cannot find host in %s profile",
+				configFile, profile)
+		}
+		config.Host = host
+	}
+
+	return nil
```

Comment:
`schema.ResourceData` does not contain required fields to fill. Is it really a positive outcome for this function?